source/github-helpers/pr-ci-status.ts

```diff
@@ -1,85 +1,81 @@
 import select from 'select-dom';
 
-import observeElement from '../helpers/simplified-element-observer';
+import * as api from './api';
 
-type CommitStatus = false | typeof SUCCESS | typeof FAILURE | typeof PENDING | typeof COMMIT_CHANGED;
-type StatusListener = (status: CommitStatus) => void;
+export const SUCCESS = Symbol('Success');
+export const FAILURE = Symbol('Failure');
+export const PENDING = Symbol('Pending');
+export type CommitStatus = false | typeof SUCCESS | typeof FAILURE | typeof PENDING;
+
+export const commitSelector = [
+	'.js-commit-group .TimelineItem--condensed', // TODO [2022-05-01]: GHE
+	'[data-test-selector="pr-timeline-commits-list"] .TimelineItem',
+].join(',');
 
-// `.TimelineItem--condensed` excludes unrelated references. See `deemphasize-unrelated-commit-references` feature
-// TODO [2022-05-01]: Remove `.js-commit-group` (GHE)
-const commitSelector = ':is(.js-commit-group, [data-test-selector="pr-timeline-commits-list"]) .TimelineItem--condensed';
+export const commitStatusIconSelector = 'details.commit-build-statuses summary .octicon';
```

Comment:
Need `summary` because the details dropdown has a list of check runs, which means lots of unrelated icons.